Source/Model/Conversation/ZMConversation+Message.swift

```diff
@@ -23,6 +23,12 @@ private let log = ZMSLog(tag: "Conversations")
 @objc 
 extension ZMConversation {
     
+    @discardableResult
+    func appendButtonAction(with id: String, referenceMessageId: UUID, nonce: UUID = UUID()) -> ZMClientMessage? {
```

Comment:
```suggestion
    func append(buttonActionWithId id: String, referenceMessageId: UUID, nonce: UUID = UUID()) -> ZMClientMessage? {
```

I think this more follows the naming convention of the other `append` methods